outputs.tf

```diff
@@ -1,24 +1,24 @@
-output "arn" {
-  value       = aws_instance.example.arn
-  description = "The EC2 instance ARN"
+output "private_subnet_nat_gws" {
+  value       = aws_nat_gateway.nat_gws
+  description = "The NAT gateways used in the private subnets in the VPC."
 }
 
-output "availability_zone" {
-  value       = aws_instance.example.availability_zone
-  description = "The AZ where the EC2 instance is deployed"
+output "private_subnets" {
+  value       = module.private.subnets
+  description = "The private subnets in the VPC."
 }
 
-output "id" {
-  value       = aws_instance.example.id
-  description = "The EC2 instance ID"
+output "public_subnets" {
+  value       = module.public.subnets
+  description = "The public subnets in the VPC."
 }
 
-output "private_ip" {
-  value       = aws_instance.example.private_ip
-  description = "The private IP of the EC2 instance"
+output "transit_gateway_id" {
+  value       = local.transit_gateway_id
+  description = "The ID of the Transit Gateway in the Shared Services account."
```

Comment:
This probably doesn't need to be an output, since it's already an output of [cisagov/cool-sharedservices-networking](https://github.com/cisagov/cool-sharedservices-networking).

Reply:
Good call- I removed the output in https://github.com/cisagov/cool-userservices-networking/pull/1/commits/ffb1e583b0b03bc42adfa499031533d4d6ff937c.